operations/onboarding.md

```diff
@@ -36,12 +36,33 @@ TOC members rotate meeting facilitation as often as we can to ensure the TOC Cha
 
 In the course of performing our duties, TOC members are expected to:
 
+* Lead or participate in at least 1 Project Due Diligence that is brought to TOC internal review status
+    * Some Due Diligence efforts may result in the TOC internal review stating the project is not ready after adopter interviews, these are still considered as meeting this expectation. Lightweight reviews/triage would not.
 * Be available to field questions and concerns from community members and redirect as needed
     *  While every TOC member is expected to do this, there may be occasions or instances where there is a lot of back and forth and the Chair or Vice Chair will coordinate and provide the response as the central contact
 * Foster discussion - ask questions, even obvious ones to move the conversation forward or to closure
 * Bring project and group conflicts that escalate to the TOC level to closure, either through elevating awareness within the TOC for discussions and consensus, direct resolution while keeping the TOC informed, or escalation to the Chair and Vice Chair as appropriate
 * Participate in meetings, presentations, etc. as needed at KCCN or other events
 
+#### Competencies
+
+In addition to the expectations defined Composition 6(b)ii and Criteria 6(d) for nomination in the CNCF Charter, TOC members should be prepared to actively employ and mature the following competencies:
+
+* Leadership (ex. transformational, democratic, servant, and visionary are leadership styles that may be used depending on the situation)
+* Deep understanding of cloud native technologies and viability for use
+* Effective communication
```

Comment:
Can we expand this to `communication - empathy, inclusivity, effectively, action-oriented` please?

Reply:
`s/effectively/effective/` 
One of those words is not like the other :)

Reply:
added!